### PR TITLE
CSS upgrade

### LINKED ISSUE #696: Upgrade CSS task
- [x] Use grunt and add SCSS.
  * Removed IE folder.
  * Moved leaflet images.
- [x] Implement ```grunt watch``` for CSS.
- [x] Start new styleguide html.
- [x] Enable Hound.
- [x] Add scss-lint.

### PR DESCRIPTION
**Pull request by xavijam**
- [x] Use grunt and add SCSS.
  * Removed IE folder.
  * Moved leaflet images.
- [x] Implement ```grunt watch``` for CSS.
- [x] Start new styleguide html.
- [x] Enable Hound.
- [x] Add scss-lint.

Fixes #696.
REVIEWER: @javierarce

### CODE REVIEW

**Comment by xavijam** [comment]
Thanks @javierarce :+1:

File: grunt/tasks/connect.js [hunk 1]
```diff
@@ -40,7 +40,18 @@ module.exports = {
             '<%= config.app %>'
           ]
         }
+      },
+      styleguide: {
+        options: {
+          port: 9001,
+          open: {
+            target: 'http://localhost:9001/themes/styleguide.html'
```
Comment (javierarce) [inline grunt/tasks/connect.js]: I think this should be 'http://localhost:9001/themes/styleguide/'

Comment (xavijam) [inline grunt/tasks/connect.js]: Yeah! thanks man!

File: grunt/tasks/watch.js [hunk 2]
```diff
@@ -7,6 +7,14 @@
 module.exports = {
   task: function() {
     return {
+      scss: {
+        files: ['themes/scss/**/*.scss'],
+        tasks: ['css', 'concat:themes', 'cssmin:themes'],
+        options: {
+					spawn: false,
```
Comment (javierarce) [inline grunt/tasks/watch.js]: Weird indentation here.

**Comment by javierarce** [comment]
This is very cool & works good too :D :+1:

**Comment by jesusvazquez** [comment]
retest this please

**Pull request merged**